CSVsToParse/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore all files in this directory to prevent accidental commits of sensitive information.
```

Comment:
Actually, the wildcard pattern `*` does ignore everything in the directory!
https://git-scm.com/docs/gitignore#_pattern_format

I debated removing the comment and let the pattern do the magic, but it may be better to err on the side of over-communciation.

Reply:
![image](https://user-images.githubusercontent.com/8202335/95431414-fa117f00-0901-11eb-9909-11448e66f738.png)
Here, you can see test files being ignored in the file tree, also printed out in the `ls` in the console, and the git status output showing that there is nothing to commit

Reply:
Awesome! Thank you.